Quadriga/src/main/webapp/WEB-INF/views/search/texts/searchTexts.jsp

```diff
@@ -1,152 +1,200 @@
+										<div class="panel panel-default">
+											<div class="panel-body">Your search has no results.</div>
+										</div>
+									</c:if>
+									<c:forEach items="${texts}" var="textfile">
+										<div class="list-group-item">
+											<p class="pull-right">
+												<a href="${textfile.refId}" title="Go to original"
+													target="_blank"> <i class="fa fa-share"></i>
+												</a>
+												<c:if test="${not empty textfile.presentationUrl}">
+													<a href="${textfile.presentationUrl}"
+														title="Go to text webpage" target="_blank"> <i
+														class="fa fa-globe"></i>
+													</a>
+												</c:if>
+											</p>
+											<h4>
+												<a href="#" data-toggle="modal" data-target="#txtModal"
+													data-txtid="${textfile.textId}"
+													data-txtname="${textfile.fileName}"
+													data-txttitle="${textfile.title}"
+													data-txtauthor="${textfile.author}"
+													data-txtdate="${textfile.creationDate}"> <i
+													class="fa fa-file-text-o" aria-hidden="true"></i> <c:if
+														test="${not empty textfile.author}">${textfile.author}, </c:if>
+													<c:if test="${not empty textfile.title}">
+														<em>${textfile.title}</em>
+													</c:if> <c:if test="${not empty textfile.creationDate}"> (${textfile.creationDate})</c:if>
+													<c:if
+														test="${empty textfile.author and empty textfile.title and empty textfile.creationDate}">No author and title information provided.</c:if>
+												</a>
+											</h4>
+
+											<p style="margin-bottom: 0px; line-height: 1.3">
+												<small>${textfile.snippet}</small>
+											</p>
+										</div>
+									</c:forEach>
+
+									<c:if test="${not empty references}">
+										<h4 style="margin-top: 30px;">External Resources</h4>
+										<c:forEach items="${references}" var="ref">
+											<div class="list-group-item">
+												<h4>
+													<a href="${ref}" target="_blank"><i class="fa fa-share"
+														aria-hidden="true"></i> ${ref}</a>
+												</h4>
+											</div>
+										</c:forEach>
+									</c:if>
+							</div> 
+
+					</div>
+				</div>
+			</div>
+		</div>
+		<div class="col-md-4" style="padding-top: 100px;">
+			<div class="row">
+				<div class="col-sm-12">
+					<c:if test="${not empty jsonstring and jsonstring != '[]'}">
+						<small>Click on a node to search for its concept.</small>
+					</c:if>
+				</div>
+				<div class="col-sm-12" style="min-height: 45px;">
+					<small>
+						<div id="searchForNode" style="display: none;">
+							Search for concept <em id="nodeName"></em>?&nbsp;&nbsp; <a
+								id="searchNodeLink"><strong>Yes, search!</strong></a>
+						</div>
+					</small>
+				</div>
+				<div class="col-sm-12">
+					<div id="networkBox"
+						style="min-height: 500px; width: 100%; text-align: left;"></div>
+				</div>
+			</div>
+		</div>
+	</div>
+</div>
```

Comment:
please don't change the indentation of this; makes it very hard to see what was changed and harder to read.